sklearn/tests/test_common.py

```diff
@@ -633,7 +633,7 @@ def test_estimators_do_not_raise_errors_in_init_or_set_params(Estimator):
     "TheilSenRegressor",
     "TransformedTargetRegressor",
     "TruncatedSVD",
-    "TweedieRegressor",
+    # "TweedieRegressor",
```

Comment:
Same comment here: please delete those lines instead of commenting them out.

Reply:
Removed 3 items: validation estimators to ignore